api-reference/50 Common/Object Structures/positionConfig/collision/collision.md

```diff
@@ -5,48 +5,127 @@ type: String | Object
 ---
 ---
 ##### shortDescription
-Specifies how to move the widget if it overflows the screen.
+Specifies how to resolve collisions between the overlay and the [boundary](/Documentation/ApiReference/Common/Object_Structures/positionConfig/#boundary) elements.
 
 ---
-A string passed to this option should contain a collision handler name, or a pair of names separated by space. If the string consists of a single collision handler name, it is applied to both horizontal and vertical axes. If you pass a pair of names separated by space, the first collision handler will be applied to the horizontal axis, the second to the vertical axis.
+Collisions occur whenever the specified position causes the overlay element to exceed the boundaries.
 
-    <!--JavaScript-->
-    // a string containing horizontal and vertical
-    // collision handlers separated by space
-    collision: "flip fit";
+You can use the following collision resolution algorithms:
 
-You can also specify different collision handlers for horizontal and vertical axes in one of the following ways.
+- *"flip"*      
+Move the overlay element to the opposite side of the [target](/Documentation/ApiReference/Common/Object_Structures/positionConfig/#of) if there is more available space there.
```

Comment:
Can we say:
Move an overlay element to the opposite side of the [target] (/Documentation/ApiReference/Common/Object_Structures/positionConfig/#of) (if space is available).

Reply:
It is important that the opposite side should provide **more** space.

Reply:
Do you mean:
Move an overlay element to the opposite side of the [target] (/Documentation/ApiReference/Common/Object_Structures/positionConfig/#of) if more space is available on that side.